kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -3159,18 +3159,19 @@ messages:
       lUsers = Send(SYS,@GetUsersLoggedOn);
       for i in lUsers
       {
+         if first_time
```

Comment:
As hinted at by the name, this is meant to be a mechanism to inform DMs that a new user has joined, kind of like an "appeal" command.  If this goes to all users, it's no longer an appeal, and thus this broadcast doesn't belong here.  You should go up higher in the stack and modify/rename the "Appeal" functions where appropriate.

Also, I don't think that DMs logging in for the first time should be broadcast.  e.g. event-specific characters that are meant to be kept secret.

Do most people want this?  As a player I think I would consider it pretty spammy.